### PR TITLE
json file: add react-native-infinite-pager

### PR DESCRIPTION
**Pull request by luism3861**
this PR add `react-native-infinite-pager` in json file.

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -8234,5 +8234,14 @@
...
     "android": false,
     "web": false
+  },
+  {
+    "githubUrl": "https://github.com/computerjazz/react-native-infinite-pager",
+    "examples": [
+      "https://snack.expo.dev/@computerjazz/infinite-pager"
+    ],
+    "ios": true,
+    "android": false,
```
Comment (Simek) [inline react-native-libraries.json:8244]:
```suggestion
```

Let's skip the `false` value. 😉

**Review by Simek — COMMENTED** [review]
Hello @luism3861, thank you for adding new entry to the directory! 👍 

I have left small suggestion, but besides that, everything LGTM!

**Review by Simek — APPROVED** [review]
I have applied the change, so we can merge and avoid the conflicts with the upcoming PR 🙂

**Comment by luism3861** [comment]
> I have applied the change, so we can merge and avoid the conflicts with the upcoming PR 🙂

thanks @Simek :)

**Pull request merged**